app/helpers.js

```diff
@@ -57,4 +57,13 @@ Helpers.getDateFromNow = function (timestamp) {
   return moment(timestamp, 'X').fromNow();
 };
 
+/**
+ * Escapes any characters that have special meaning in JQuery selectors.
+ * @param {string} selector
+ * @return {string}
+ */
+Helpers.escapeJQuerySelector = function (selector) {
```

Comment:
It handles CSS metachars and escapes them with `\` instead of `%`.
The doc ID is used as ID for some `<div>`. When the ID has chars like `#` or `.`,  JQuery fails because it considers those metachars. E.g. if you want to find element with ID `#foo.bar` your query would be `$('#\#foo\.bar')`